external-crates/move/crates/move-vm-runtime/src/interpreter.rs

```diff
@@ -273,22 +255,20 @@ impl Interpreter {
                         .map_err(|e| set_err_info!(current_frame, e))?;
 
                     if func.is_native() {
+                        let func_clone = func.clone();
```

Comment:
there may be a good reasons here but why do we clone just so we pass a reference?

Reply:
`call_native` takes the `func` by value, and we need to call into the tracer after the call has finished, so we need to clone it first. We could pass this by reference to `call_native` however it's an `Arc` so super super cheap "copy", and we'll only incur this in tracing builds (otherwise it should be optimized away).